src/bio.c

```diff
@@ -1378,8 +1378,11 @@ long wolfSSL_BIO_get_mem_ptr(WOLFSSL_BIO *bio, WOLFSSL_BUF_MEM **ptr)
             if (closeFlag == BIO_CLOSE)
                 wolfSSL_BUF_MEM_free(bio->mem_buf);
 
+        if ((closeFlag < 0) || (closeFlag > 255)) {
```

Comment:
closeFlag is WOLFSSL_BIO_NOCLOSE or WOLFSSL_BIO_CLOSE only.

Reply:
Updated